### PR TITLE
SW-2447 Header Spacing Fix

### PR DESCRIPTION
**Pull request by charlie-tf**
- fix header padding on monitoring page

image

### CODE REVIEW

File: src/components/Monitoring/index.tsx [hunk 1]
```diff
@@ -152,7 +152,7 @@ export default function Monitoring(props: MonitoringProps): JSX.Element {
       <Grid container>
         {hasSeedBanks ? (
           <>
-            <PageHeaderWrapper nextElement={contentRef.current}>
+            <PageHeaderWrapper nextElement={contentRef.current} nextElementInitialMargin={-24}>
```
Comment (karthikbtf) [inline src/components/Monitoring/index.tsx:155]: may want to skip the negative margin on mobile, it seems to be blending into the seedbank picker

File: src/components/Monitoring/index.tsx [hunk 2]
```diff
@@ -37,11 +37,11 @@ const useStyles = makeStyles((theme: Theme) => ({
   titleRootContainer: {
     display: 'flex',
     alignItems: 'center',
+    paddingBottom: theme.spacing(4),
```
Comment (karthikbtf) [inline src/components/Monitoring/index.tsx:40]: ideally margin to allow for space between components, padding if we want space inside the component so we can draw borders etc..

**Review by karthikbtf — APPROVED** [review]
looks good, a few comments

File: src/components/Monitoring/index.tsx (hunk 1 above)
Comment (charlie-tf) [reply src/components/Monitoring/index.tsx:155]: Left it in, and added the margin to the header for mobile. It looks correct now.

**Review by karthikbtf — APPROVED** [review]

File: src/components/Monitoring/index.tsx (hunk 2 above)
Comment (charlie-tf) [reply src/components/Monitoring/index.tsx]: made the change. Thanks for the feedback!

**Pull request merged**